src/ModuleManager.php

```diff
@@ -57,27 +55,49 @@ private static function getPackageSubscriptions(PackageInterface $package): arra
      * @param PackageInterface $package Composer package containing this module.
      * @throws ModuleAlreadyExistsException if the module is already installed.
      * @throws ClassNotFoundException if an event subscription class cannot be resolved.
-     * @throws WrongTypeException if an event subscription class does not implement the EventHandler
+     * @throws WrongTypeException if an event subscription class does not implement the Module
      * interface or the pattern is not a valid regular expression.
      */
     public function register(string $library, PackageInterface $package)
     {
-        $m = $this->g->getEntityManager()->getRepository(Module::class)->find($library);
+        $m = $this->g->getEntityManager()->getRepository(\LotGD\Core\Models\Module::class)->find($library);
         if ($m) {
             throw new ModuleAlreadyExistsException($library);
         } else {
             // TODO: handle error cases here.
-            $m = new Module($library);
+            $m = new \LotGD\Core\Models\Module($library);
```

Comment:
Why the FQCN instead of use? If it is in case of name clashing, you could also "use ... as ..."